src/command_modules/azure-cli-vsts/azure/cli/command_modules/vsts/custom.py

```diff
@@ -118,14 +125,16 @@ def _call_rp_configure_cicd(
     :type resource_group_name: String
```

Comment:
those are two different parameters, I'm still passing resource_group_name, what I changed is registry_resource_group-->registry_resource_id